storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -1419,6 +1419,25 @@ public class Config extends HashMap<String, Object> {
      */
     @IsString
     public static final String STORM_ZOOKEEPER_TOPOLOGY_AUTH_SCHEME = "storm.zookeeper.topology.auth.scheme";
+
+    /** Enable SSL/TLS for ZooKeeper client connection. */
+    @IsBoolean
+    public static final String ZK_SSL_ENABLE = "storm.zookeeper.ssl.enable";
```

Comment:
Can we rename those constants starting with `ZK` to be consistent with the rest of the constant names? They all are something like `STORM_ZOOKEEPER_XY`?